index.html

```diff
@@ -10,951 +12,1067 @@
+			<section class="section-heading has-text-centered" aria-label="About us">
+				<h1 class="title is-2">About Us</h1>
+				<h2 class="subtitle is-5">We of all trades, master of "some"</h2>
+				<div class="container">
+					<p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
+						field of
+						<b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
+						Looking for an opportunity to work and upgrade, as well as being involved in an organization that
+						believes
+						in gaining a competitive edge and giving back to the community.</p>
+				</div>
+			</section>
 
-  <div class="section" id="about">
-    <div class="section-heading has-text-centered">
-      <div class="title is-2">About Us</div>
-      <div class="subtitle is-5">We of all trades, master of "some"</div>
-      <div class="container">
-        <p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
-          field of
-          <b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
-          Looking for an opportunity to work and upgrade, as well as being involved in an organization that
-          believes
-          in gaining a competitive edge and giving back to the community.</p>
-      </div>
-    </div>
+			<div id="mainsection" class="content container">
+				<section aria-label="Hello World">
+					<h2 class="is-1 title">Hello World</h2>
+					<p>Lorem ipsum<sup><a tabindex="0" role="link">[1]<span class="visuallyhidden">&nbsp;footnote in dictionary</span></a></sup> dolor sit amet, consectetur adipiscing elit. Nulla accumsan, metus ultrices eleifend gravida, nulla nunc varius lectus, nec rutrum justo nibh eu lectus. Ut vulputate semper dui. Fusce erat odio, sollicitudin vel erat vel, interdum mattis neque. Sub<sub>script</sub> works as well!</p>
+				</section>
+				<section aria-label="Second level">
+					<h2 class="is-2 title">Second level</h2>
+					<p>Curabitur accumsan turpis pharetra <b>augue tincidunt</b> blandit. Quisque condimentum maximus mi, sit amet commodo arcu rutrum id. Proin pretium urna vel cursus venenatis. Suspendisse potenti. Etiam mattis sem rhoncus lacus dapibus facilisis. Donec at dignissim dui. Ut et neque nisl.</p>
+					<ul>
+						<li>- In fermentum leo eu lectus mollis, quis dictum mi aliquet.</li>
+						<li>- Morbi eu nulla lobortis, lobortis est in, fringilla felis.</li>
+						<li>- Aliquam nec felis in sapien venenatis viverra fermentum nec lectus.</li>
+						<li>- Ut non enim metus.</li>
+					</ul>
+				</section>
+				<section aria-label="Third level">
+					<h3 class="is-3 title">Third level</h3>
+					<p>Quisque ante lacus, malesuada ac auctor vitae, congue <a tabindex="0" role="link">non ante<span class="visuallyhidden">&nbsp;read more</span></a>. Phasellus lacus ex, semper ac tortor nec, fringilla condimentum orci. Fusce eu rutrum tellus.</p>
```

Comment:
the tag **<a** still has the `role="link"` by default

Reply:
The same comment as previous